OrbitGl/TimeGraph.cpp

```diff
@@ -588,7 +589,7 @@ void TimeGraph::UpdatePrimitives(bool a_Picking) {
 
                 textBox.SetText(text);
               } else if( timer.m_Type == Timer::INTROSPECTION) {
-                textBox.SetText(timer.EncodedString());
+                textBox.SetText(GStringManager->Get(timer.m_UserData[0]));
```

Comment:
Maybe pass a pointer to the StringManager in c-tor, or add a setter? I think we should try to reduce globals in the code as much as possible.

Reply:
Is this a general argument against globals or do you have a specific concern about this here? Would you prefer if the string manager were a singleton? I think there's value in having objects that are "resource managers" easily accessible from anywhere in the application (avoids lots of plumbing to various places).

Reply:
This is general argument against globals. According to C++ Style Guide: https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables  "More formally it means that the type has no user-defined or virtual destructor and that all bases and non-static members are trivially destructible", we have a flat_hash_map there... I do not think flat_hash_map is trivially destructible.

Reply:
Ok, will make this a member of OrbitApp.